readme.md

```diff
@@ -363,13 +363,15 @@ test(t => {
 });
 ```
 
-You can also use your own local Babel version:
+To use your own local Babel version, specify the version you want in your package.json, and ensure you're using npm@3 (or run `$ npm dedupe`), so the dependency will be deduped.
```

Comment:
> Since we depend on a loose Babel 5 version, the user can set an exact version and it will be deduped to the user's choice.

This just doesn't seem very relevant to me. AVA uses Babel, fine. The version may change, even in patch releases, as long as the test behavior remains the same. Why should a user be concerned with setting an exact version in the project in order to influence the version AVA uses?

What I find more important as a user is knowing that I can install whatever Babel version I like and it doesn't break AVA.